### ISSUE TITLE
Missing Symbols for Elsa 2.11.0 on nuget.org

### ISSUE THREAD

**Issue by holidayfun** [elsa-workflows/elsa-core#3820]
## Summary

There are currently no symbol packages available on nuget.org, even though they should be published by the [Build and Publish Elsa Packages](https://github.com/elsa-workflows/elsa-core/actions/workflows/publish-latest-elsa.yml) action.

## Explanation

I've noticed that debugging the most recent version of Elsa using the Nuget symbol server is not possible. When the feedz.io symbol server is added in VS, stepping into Elsa code works as expected.

I've then looked into the [actions for Elsa 2.11.0](https://github.com/elsa-workflows/elsa-core/actions/runs/4478173013) and noticed the "Publish to Nuget" step failed due to the following:

`error: Response status code does not indicate success: 409 (A package with ID 'Elsa' and version '2.11.0' already exists and cannot be modified.)`

So i guess the packages were previously published (by hand?) without the symbol packages. Personally i think the packages should be published by the github action only, but if manually published, can we at least publish the symbol packages as well?

## Notes

You can tell if the symbol packages are available on nuget.org, e.g. for another project the following info is available (see the `Download symbols` link). 

![grafik](https://user-images.githubusercontent.com/3428158/226859703-a73ed9e0-b516-41ef-ad4d-c06e5ee3123b.png)

**Comment by sfmskywalker** [comment]
Hi @holidayfun , indeed for this release I uploaded the packages manually via the CLI. I also pushed the symbols package, but NuGet reported it could not validate the symbols packages. I have yet to look into that issue, but if you have any suggestions I'd be happy to hear them. If you want, I can share the packages + symbols if you want to take a look.

**Comment by holidayfun** [comment]
Do you know the error message you got from nuget by any chance?

**Comment by holidayfun** [comment]
I've taken a quick look into it and i think i found the culprit.

According to https://github.com/jbevain/cecil/issues/610 Cecil does not generate a PdbChecksum entry in the assembly. And Fody uses Cecil.

I tried packing and uploading the Elsa.Core package (with another package id) as is and could reproduce the error you mentioned:
![grafik](https://user-images.githubusercontent.com/3428158/226891746-1c077efb-7c7a-4ca6-b573-3fec08a3ecbd.png)

To confirm my theory i removed the `<Import Project="..\..\..\configureawait.props" />` entry from the project file and retried uploading to nuget, this time successfully publishing the symbol package.

![grafik](https://user-images.githubusercontent.com/3428158/226892966-a0cc69df-3e3f-47c1-b9e2-72ee9f4b830f.png)


So as far as i can tell, publishing symbol packages to nuget will continue to fail due to the processing with Fody and the validation of symbol packages.

**Comment by holidayfun** [comment]
I've looked into it again and found that https://github.com/jbevain/cecil/pull/810 should have fixed this behavior.

The current version of Fody used is 6.6.0, which does not include the fix above.

I've tried updating to the most recent version of Fody 6.6.4 and was able to successfully publish the symbol packages to nuget.

Should i create a pull request for updating the Fody version in `configureawait.props`?
